packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,72 @@
+# Dropdown
+
+This is the Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```twig
+<Button data-toggle="dropdown" data-target="DropdownExample">Open Dropdown</Button>
+<Dropdown id="DropdownExample">Dropdown Content</Dropdown>
+```
+
+Usage with link:
+
+```twig
+<ButtonLink href="javascript:void(0)" data-toggle="dropdown" data-target="CollapseExample">Open Dropdown</ButtonLink>
+...
+```
+
+Responsive usage for tablet
+
+```twig
+...
+<Button class="d-tablet-none">Open Dropdown</Button>
+<Dropdown id="DropdownExample" breakpoint="tablet">Dropdown Content</Dropdown>
+```
+
+Advanced example usage:
+
+```twig
+<Button data-toggle="dropdown" data-target="DropdownExample">Open Dropdown</Button>
+<Dropdown elementType="span" id="DropdownExample" class="custom-dropdown-class">Dropdown Content</Dropdown>
+```
+
+Without lexer:
+
+```twig
+{% embed "@spirit/dropdown.twig" with { props: {
+    breakpoint: 'tablet',
+    class: 'dropdown-custom-class',
+    elementType: 'span'
+}} %}
+    {% block content %}
+          Dropdown content
+    {% endblock %}
+{% endembed %}
+```
+
+You can add any custom trigger like `<button>` or `<a>` but it is necessary to add `data-toggle="dropdown"`, `data-target="<id>"`
+attributes to register trigger events.
+
+## API
+
+| Prop name     | Type     | Default | Required | Description                              |
+| ------------- | -------- | ------- | -------- | ---------------------------------------- |
+| `id`          | `string` | -       | yes      | Dropdown ID                              |
+| `breakpoint`  | `string` | -       | no       | Breakpoint level [mobile,tablet,desktop] |
+| `class`       | `string` | `null`  | no       | Custom CSS class                         |
```

Comment:
The `fullWidth` option is missing from the API.

Reply:
Added as `isFullWidth`.